contracts/upgradeable_contracts/RTokenConnector.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-solidity/contracts/AddressUtils.sol";
+import "../interfaces/IRToken.sol";
+import "./Ownable.sol";
+import "./ERC20Bridge.sol";
+
+contract RTokenConnector is Ownable, ERC20Bridge {
+    bytes32 internal constant RTOKEN = 0xdce7cb71f7466970418465f97f1fb7d721c1661bc59b1f0caa7d1f8ce374a47e; // keccak256(abi.encodePacked("rToken"))
+
+    function initializeRToken(
+        address _rToken,
+        address[] _recipients,
+        uint32[] _proportions
+    ) external onlyOwner returns (uint256) {
+        require(AddressUtils.isContract(_rToken));
+        addressStorage[RTOKEN] = _rToken;
+        erc20token().approve(_rToken, uint256(-1));
+        return createAndChangeRTokenHat(_recipients, _proportions);
```

Comment:
for this and all following cases where it is applicable will the corresponding method revert in case of an error or will return `false`?

Reply:
It will revert. And I understood you, the return statement will be removed

Reply:
why do we need to return anything if there is no way to get result of transaction except its status?

Reply:
Sorry, I do not quite understand you. Why is there no way to get result of transaction? And do you just want to remove all returned values?

Reply:
how do you get a result of a method call if it is executed by `sendRawTransaction`?

Reply:
There is no way to get it I think. I will remove returned values